app/tests/features/steps/steps.py

```diff
@@ -71,3 +73,18 @@ def step_impl8(context):
     convert_msg = convert_queue.fetch_message()
     assert stream_msg
     assert convert_msg
+
+
+@then("run mseed conversion from command line")
+def step_impl(context):
+    convert_msg: QueueMessage = convert_queue.fetch_message()
+    message_content: dict = json.loads(convert_msg.content)
+    azure_storage_decimated_file_path: str = ""
+    file_format: str = "";
+    for key, value in message_content.items():
+        if (key == "path"):
+            azure_storage_decimated_file_path = value
+        if (key == "format"):
+            file_format = value
```

Comment:
```suggestion
    azure_storage_decimated_file_path: str = message_content["path"]
    file_format: str = message_content["format"]
```
Cumbersome way of getting dict values.
Also, no need to check for values, if they are not there, the test should fail.